### DISCUSSION TITLE
Configuration options in Long view mode

### DISCUSSION THREAD

**Discussion by mnpqraven** [leo-arch/clifm#141] [Q&A] [answered]
Hello, is there any configuration options for long view mode .e.g showing/hiding specific columns ? I personally use the long view alot as all the items are placed on where I'm always looking at, but I don't really need the uid/gid column and in small windows they'll make the lines wrap. Is there a command to show/hide the uid/gid column ?

**Comment by leo-arch** [comment]
Hi @mnpqraven. Thanks for reporting!

I'll take a look at it. For sure it seems a reasonable request.

I'll let you know as soon as I have this thing working, at least partially.

And no, there's currently no option to customize properties fields in long view mode.

**Comment by leo-arch** [comment, marked as answer]
Done. The `PropFields` option in the [config file](https://github.com/leo-arch/clifm/blob/master/misc/clifmrc) allows you to turn on/off properties fields in long view mode.

Clone and Install the last version, and update your config file, or just add the line manually:

```sh
PropFields="aits"
```
to displays all fields (default).

Or, to exclude user/group IDs:

```sh
PropFields="ats"
```

Take a look at [the docs](https://github.com/leo-arch/clifm/wiki/Common-Operations#longdetail-view-mode) for more information.

**Comment by leo-arch** [comment]
Hi @mnpqraven. The first implementation of this new feature used `a` (for attributes) as a representation of the first field of the file's properties. But since `attributes` is not the proper name for this information, but rather `permissions`, `a` was replaced by `p`. So, to display all properties fields now you should use `pits` instead of `aits`.

`a` is still supported, but will be removed before the next release (1.7). Please update the program and your config before the next release.

**Reply by mnpqraven** [reply]
Roger that, I've update the config. Thanks for the quick work leo!

**Comment by leo-arch** [comment]
Support for `a` in `PropFields` option already removed.

**Comment by leo-arch** [comment]
Now you can use `n` instead of `p` in `PropFields` to use numeric/octal notation for file permissions instead of symbolic. Using `n` produces shorter output (only 4 chars) than `p` (14 chars!).

**Comment by leo-arch** [comment]
The `t` switch has been removed. The time stamp switch now takes three possible values: `a`, `c`, or `m`, i.e. last (a)ccess, (m)odification, or status (c)hange time. See the [updated documentation](https://github.com/leo-arch/clifm/wiki/Common-Operations#file-details).

The new default is `pims`, or, for your setup, `pms`.
